### PR TITLE
Updated ansible scripts to use service for mythra and chrome driver

### PR DESCRIPTION
**Pull request by MeNsaaH**
- Updated nginx configuration
- switch to using service files to start and stop mythra and chromedriver
- Use github api to get latest release

### CODE REVIEW

**Comment by deven96** [comment]
You'd have to effect the service type changes on the gcp yaml as well on [mythra](https://github.com/deven96/mythra/blob/master/.github/workflows/gcp.yml)

File: ansible/mythra.yml [hunk 1]
```diff
@@ -17,70 +17,84 @@
...
       debug:
         msg: "{{ latest_tag }}"
 
     - name: Download latest tag
-      get_url: 
-        url: "{{ 'https://github.com/deven96/mythra/releases/download/{}/mythra-ubuntu-18.04-{}'.format(latest_tag.stdout, latest_tag.stdout) }}"
-        dest: "{{ '{}/mythra'.format(mythra_web_path) }}"
+      tags: mythra
+      get_url:
+        url: "{{ 'https://github.com/deven96/mythra/releases/download/{}/mythra-ubuntu-18.04-{}'.format(latest_tag, latest_tag) }}"
```
Comment (deven96) [inline ansible/mythra.yml:53]: does using "latest_tag" just work like that without specifying stdout?

Comment (MeNsaaH) [reply ansible/mythra.yml:53]: I've registered a new variable `latest_tag` using `set_fact` that gets it's value from the json object returned by github api

**Comment by deven96** [comment]
LGTM

**Pull request merged**